src/js/components/CheckBox/CheckBox.js

```diff
@@ -215,7 +215,6 @@ const CheckBox = forwardRef(
         reverse={reverse}
         {...removeUndefined({ htmlFor: id, disabled })}
         checked={checked}
-        label={label}
```

Comment:
can we add this back in and maybe have `labelProp={label}`

Reply:
if we change this to `labelProp` we will also need to change 
```
typeof props.label

```
to 
```
typeof props.labelProp

```

https://github.com/grommet/grommet/blob/master/src/js/components/CheckBox/StyledCheckBox.js#L57